client/utils/history.tsx

```diff
@@ -61,7 +61,7 @@ const getHistoryRowElement = (text, historyItem, users) => {
     if (text) {
         return (
             <div>
-                <span><strong>{text}</strong>{gettext(' by ')}</span>
+                <span><strong>{text}</strong>{historyItem.user_id != null ? gettext(' by ') : null}</span>
```

Comment:
What was the issue with the history component?

Reply:
it would be like `Ingested by Monday ...` when something was done without user action